card/mod/core/set/all/assign_attributes.rb

```diff
@@ -35,6 +35,9 @@ def extract_subcard_args! args
 def prepare_assignment_params args
   params = ActionController::Parameters.new(args)
   params.permit!
+  if params[:db_content].is_a? Array
+    params[:db_content] = params[:db_content].join("\n")
+  end
```

Comment:
It was already allowed because Rails automatically casts everything to the right type. But now it will be casted to something sensible (`[1, 2]` to `"1\n2"` instead of `"[1, 2]"`). Should have happened a long time ago in my opinion. Initializing pointer cards was always a mess.